sdk/python/feast/driver_test_data.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 from pytz import FixedOffset, timezone, utc
 
-from feast.infra.provider import DEFAULT_ENTITY_DF_EVENT_TIMESTAMP_COL
+from feast.infra.utils.common_utils import DEFAULT_ENTITY_DF_EVENT_TIMESTAMP_COL
```

Comment:
This is a bit of a nitpick, but whenever I see a "utils" folder/file/class then I ask whether we are actually adding to technical debt by creating a place where random code can be thrown. Is it really unavoidable that we create a `common_utils`? Is there any way for us to be more descriptive?

Reply:
I started utils under infra to put cloud-specific utility functions in the same place (e.g. aws_utils). I added common_utils as a place for utility methods that multiple infra components can use (in this case BigQuery and Redshift). Do you have any suggestions on how to change this for the better?

Reply:
https://github.com/feast-dev/feast/blob/master/sdk/python/feast/infra/offline_stores/helpers.py?

Reply:
> utility methods that multiple infra components can use (in this case BigQuery and Redshift)

It seems more like you are referring to offline stores than any arbitrary infra.

Reply:
I'd probably just make it `offline_store_util.py`